app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      before_action :authenticate_api_v1_user!, only: %i[index create]
+      def index
+        render_paginated Rent.filter(params.slice(:user_id, :book_id))
+      end
+
+      def create
+        rent = Rent.new(
+          book_id: params[:book_id],
+          user_id: params[:user_id],
+          from: params[:from],
+          to: params[:to]
+        )
```

Comment:
this is ok, but a cool way to do this is something like `rent = Rent.new(permitted_params)` where permitted params is a private method where you do
`params.require(:rent).permit(:book_id, :user_id, :from, :to)`
The json for that request would be `{ "rent" : { ... } }`, which is also nice.